### ISSUE TITLE
Win10, Defold doesn't start at all

### ISSUE THREAD

**Issue by ImaginaryStargazer** [defold/defold#8341]
**Describe the bug (REQUIRED)**
No matter which version I try from the list below, Defold just doesn't start on my system.
Trying to run it from terminal yields no output, while the log file in `~\AppData\Local\Defold` folder only writes variations of this line:

```ini
2023-12-20 18:49:48.248 523  [JavaFX-Launcher] WARN  javafx - Unsupported JavaFX configuration: classes were loaded from 'unnamed module @5b399b10'
```

At first I thought that those `unnamed module @yada-yada` were all different, but actually there were only 4 of them so far:
- `@42fed190`
- `@4bb81afc`
- `@5b399b10`
- `@762fe7d0`

Also note, that while versions ≥ 1.4.8 weren't launching anything, version 1.4.7 actually showed a splash screen, but hanged there forever without opening the editor (for like an hour, before I forcefully stopped it).

Maybe, the culprit lies in the fact that I have Java 8, 11, 17 and 21 installed on the system, with 21 being in JAVA_HOME:

![image](https://github.com/defold/defold/assets/45818008/9804c920-8059-4f15-8547-429171114128)

But skimming through other issues in this repo I've seen people with different setups get this log message, so maybe not.

<p>‎ </p>
<p>‎ </p>

**To Reproduce (REQUIRED)**
1. Download and extract, say, latest version of Defold 
2. Double-click to run the `Defold.exe` or run it from command-line.

**Expected behavior (REQUIRED)**
Defold's GUI should open.

**Defold version (REQUIRED):**
  - Version 1.6.2 stable
  - Version 1.6.3 beta
  - Version 1.4.8 stable
  - Version 1.4.7 stable
  - Version from the Steam launcher

**Platforms (REQUIRED):**
 - OS: Windows 10 Home, 22H2, OS Build 19045.3803

**P.S.:** I would still like to try out the editor, so I'll probably spin up a Linux VM in VirtualBox and try it there. But not being able to run it from Windows 10 natively kinda sours the experience just a little :)

**Comment by britzl** [comment]
> ```ini
> WARN  javafx - Unsupported JavaFX configuration: classes were loaded from 'unnamed module
> ```

This is just a warning and is no indication of why the editor doesn't start in your case.

> Maybe, the culprit lies in the fact that I have Java 8, 11, 17 and 21 installed on the system, with 21 being in JAVA_HOME

Nope. The editor comes bundled with a Java version and should completely ignore whatever Java versions you may have installed on the system.

> I would still like to try out the editor

Let's try to figure this out!

1. Let's start by checking if the editor writes any log file. Please read this and then check if you have a log. If you have a log file please share it here. https://defold.com/manuals/editor/#editor-logs
2. Are you using an anti-virus software of some kind which might block the editor from starting? The Defold.exe and other files are code signed and should bypass Windows Smart Screen, but who knows, you might have security software which incorrectly classifies the Defold.exe as a threat. [We've had a similar thing with McAfee](https://forum.defold.com/t/luajit-32-exe-problem-on-windows-with-mcafee-antivirus/73890/26?u=britzl) a while back but that got resorted when they whitelisted the file

**Comment by ImaginaryStargazer** [comment]
@britzl Sure, let's do it!

1. Yes, it writes, but only one line I've mentioned in the issue. Each time I try to launch the 1.6.2 version from Steam or from an unpacked zip folder from website downloads it simply writes that warning line to the file, without errors or anything of sorts.

File: `C:\Users\mutal\AppData\Local\Defold\editor2.2023-12-21.log`
Contents:

```ini
2023-12-21 16:08:58.075 636  [JavaFX-Launcher] WARN  javafx - Unsupported JavaFX configuration: classes were loaded from 'unnamed module @4bb81afc'
2023-12-21 16:10:11.470 482  [JavaFX-Launcher] WARN  javafx - Unsupported JavaFX configuration: classes were loaded from 'unnamed module @4bb81afc'
```

2. Nope, only the default Windows Security Center. Moreover, I've added the folder with Defold to it's exceptions and that didn't affect anything.

**Comment by britzl** [comment]
Ok, what if you run Defold.exe from a command prompt? What is the output?

**Comment by ImaginaryStargazer** [comment]
> Ok, and from where are you running the Defold.exe? From somewhere in user home?

I've tried both:
1. Running Defold.exe right in the folder with it.
2. Running Defold.exe from user's home folder.

Neither produces anything in terminal:
![image](https://github.com/defold/defold/assets/45818008/62cb21e2-201e-4a8d-8c88-6ba2ecedfb27)

**Comment by britzl** [comment]
Do you perhaps have any clues in the Windows Events Log? Check using the Windows Event Viewer tool.

**Comment by ImaginaryStargazer** [comment]
No, no mentions of Defold and no other logs reported at times of trying to launch the Defold.exe in Application, Security or System logs.

**Comment by britzl** [comment]
I'm at a loss then. I'm thinking if it is GPU driver related, but I guess I'd expect you to get further and more logs to be written.

**Comment by jeffnyman** [comment]
For what it's worth, and if it helps to get another data point: I get exactly the same issue as the OP although mine is on Windows 11 Pro using an NVIDIA GeForce RTX 3080.

I get the exact same "editor2.2024-04-05.log" in the AppData location with the same JavaFX warning. Running as administrator has no effect. Running from the icon in Explorer or running from the command line has no effect. Nothing is listed in the Event Viewer.

I use Norton so I made sure to exclude the directory where Defold is running. This had no effect. I also just disabled Norton entirely. That, too, had no effect.

I tried this with the Steam version as well as the version loaded from the main web site and the same exact thing occurs.

**Comment by AGulev** [comment]
Looks like similiar issue  https://forum.defold.com/t/defold-not-starting-on-windows-11/74284/66

**Comment by AGulev** [comment]
we expect it to be solved in 1.13.0
